code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm

```diff
@@ -167,6 +167,29 @@ Difficulty: Hard
 	var/crate_type = pick(loot)
 	var/obj/structure/closet/crate/C = new crate_type(loc)
 	new core_type(C)
+	if(!enraged)
+		return
+	if(enraged)
+		for(var/mob/living/M in urange(40, src)) //Bigger range, ran once per shift, as people run away from vetus as it blows up.
+			if(M.client)
+				new /obj/item/disk/fauna_research/vetus(C)
+
+/mob/living/simple_animal/hostile/megafauna/ancient_robot/enrage()
+	. = ..()
+	armour_penetration_percentage = 66
+	TL.armour_penetration_percentage = 66
+	TR.armour_penetration_percentage = 66
+	BL.armour_penetration_percentage = 66
+	BR.armour_penetration_percentage = 66
+
+/mob/living/simple_animal/hostile/megafauna/ancient_robot/unrage()
+	. = ..()
+	armour_penetration_percentage = 50
```

Comment:
These seem like magic numbers, can you make them defines?

Reply:
I could? Defining 66% and 50% seems strange, but I can if you want?

Reply:
Whatever you prefer, they only seem to be used here so it doesnt really matter, just something I noticed